docs/contributing/coding-standards/components.md

```diff
@@ -10,19 +10,19 @@ Generally, folder and file names should be singular, for example ‘accordion’
 
 When creating your component, you should create the following files in the component’s folder:
 
-      - `README.md`
-      - `_[component-name].scss`
-      - `_index.scss`
-      - `[component-name].yaml`
-      - `macro.njk`
-      - `template.njk`
-      - `template.test.js`
+- `README.md` - Light documentation pointing to installation instructions and component documentation on <https://design-system.service.gov.uk/>
+- `_[component-name].scss` - An SCSS file to generate the styles just for the component, importing the variables, functions and mixins from `../../base` and the styles of the component (`./index.scss`).
```

Comment:
```suggestion
- `_[component-name].scss` - An SCSS file to generate the styles for the component, import the variables, functions and mixins from `../../base` and the styles of the component (`./index.scss`).
```

Reply:
I think this edit loses the meaning I was trying to express here, to help distinguish this file from `_index.scss`. This file is used so that people using the `govuk-frontend` package can [import the styles of a single component on its own](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#import-an-individual-component-s-css-using-a-single-import). 

Re-reading it, it feels like it's not really relevant how it does it (the part about importing the variables,...) as a developer will see that pretty clearly when they open the file, and there may be other parts getting added later on that'd require us to update that doc.  Maybe just that it delegates the actual styling to the `_index.scss` file to give people a heads up that there's not much going on in that file.

Maybe something like:
> An SCSS file to generate the styles for this component only. It delegates the CSS generation to the `_index.scss` file (see right after).

Reply:
Ah yes, that is very clear. I don't think the bracket info at the end is needed but I'll amemd the suggested edit to your text.